doc/dev/release.md

```diff
@@ -128,6 +128,23 @@ These steps describe how to conduct a release of the SDK, upgrading from `v1.2.0
 
 **Note:** `master` should be frozen between steps 1 and 3 so that all commits will be either in the new release or have a pre-release version, ex. `v1.2.0+git`. Otherwise commits might be built into a release that shouldn't or have an incorrect version, which makes debugging user issues difficult.
 
+### (Before beginning) Updating the operator-sdk-samples repo
```

Comment:
Is there a chicken and egg problem with this? Do the sample operator dependencies need to reference the latest SDK release?

Reply:
Yes to some extent. We’d have to break one of the two repo’s mid-release, so I figured it would be easier to break the samples for a short period of time than the SDK. WDYT?

Reply:
I was naively thinking we'd update `operator-sdk`, then go back and fix `operator-sdk-samples`, and finally tag `operator-sdk-samples`.

But I wasn't thinking about docs and links in `operator-sdk` that reference samples at that tag. We could do something similar to [this](https://github.com/operator-framework/operator-sdk/blame/master/README.md#L57), where:
* In SDK links, we always point to `operator-sdk-samples` at `master`
* In `operator-sdk-samples` README.md, we tell users to use the appropriate tag.

If we go the route of updating samples first, we wouldn't be able to compile and run the new samples with the new version since the new version wouldn't be released.

So we'd have to make the changes without knowing if they'd work? And then if anything ends up being broken in `operator-sdk-samples` after the release, we'd go back to `operator-sdk-samples`. fix what's broken, and move the tag?

Reply:
I was thinking we'd use an SDK commit just before the release, but that's not great because the operator wouldn't be based off of that release. Linking to samples `master` is hard because code changes and line numbers will be off, but a big **NOTE** at the top of the README.md should be good enough to point users in the right direction.

We should also update the SDK release notes after releasing the samples repo with a **NOTE** pointing to the same samples repo tag.

Reply:
👍 on linking to the operator-sdk-samples master and then letting users pick the appropriate release tag from there. It's better if we have the SDK release process decoupled from the samples repo for the most part. And not having to do that before an SDK release.

Reply:
Done.